ansible/roles/k8s_cert_manager/README.adoc

```diff
@@ -0,0 +1,57 @@
+== Role: Kubernetes Certificate Manager
+
+Deploy Kubernetes Certificate Manager on kubernetes cluster
+
+=== Description
+
+This role will install the Kubernetes certificate manager on a kubernetes cluster.
+
+It also installes the _Godaddy webhook_.
```

Comment:
`It also installes` -> `Remark`: it also installs the Godaddy webhook

Reply:
fixed